Shoko.Server/Scheduling/Jobs/Trakt/CheckTraktTokenJob.cs

```diff
@@ -41,9 +41,11 @@ public override Task Process()
             // Check if the token needs refreshing
             if (ForceRefresh || DateTime.Now.Add(TimeSpan.FromDays(45)) >= expirationDate)
             {
-                _traktHelper.RefreshAuthToken();
-                var newExpirationDate = DateTimeOffset.FromUnixTimeSeconds(long.Parse(settings.TraktTv.TokenExpirationDate)).DateTime;
-                _logger.LogInformation("Trakt token refreshed successfully. New expiry date: {Date}", newExpirationDate);
+                if (_traktHelper.RefreshAuthToken())
+                {
+                    var newExpirationDate = DateTimeOffset.FromUnixTimeSeconds(long.Parse(settings.TraktTv.TokenExpirationDate)).DateTime;
+                    _logger.LogInformation("Trakt token refreshed successfully. New expiry date: {Date}", newExpirationDate);
+                }
```

Comment:
We might want to log when it fails to refresh too.

Reply:
I removed that one because we have the catch statement that catches the error ? But I can add something again if you want.